pkgs/development/libraries/libxmlxx/default.nix

```diff
@@ -16,11 +15,18 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = [ libxml2 glibmm ];
 
+  passthru = {
+    updateScript = gnome3.updateScript {
+      packageName = pname;
+    };
+  };
+
   meta = with stdenv.lib; {
-    homepage = http://libxmlplusplus.sourceforge.net/;
+    homepage = "http://libxmlplusplus.sourceforge.net/";
     description = "C++ wrapper for the libxml2 XML parser library";
     license = licenses.lgpl2Plus;
     platforms = platforms.unix;
     maintainers = with maintainers; [ phreedom ];
+    repositories.git = "https://gitlab.gnome.org/GNOME/libxmlplusplus/";
```

Comment:
```suggestion
```

Think we're good for merge after that

Reply:
We've got `meta.repositories.git` in the tree 74 times.
I think if the repo is neither at `src.url` or `meta.homepage` it's okay to have.

Reply:
see https://github.com/NixOS/nixpkgs/issues/61435. if ratified perhaps we can use it widespread.

Reply:
Alright, thanks for creating the issue.